src/adapters/IDB.js

```diff
@@ -116,11 +116,22 @@ export default class IDB extends BaseAdapter {
   /**
    * Constructor.
    *
-   * @param  {String} dbname The database nale.
+   * Options:
+   * - `{String}` `storage` The storage to use: "persistent" or "temporary".
+   *
+   * If storage is specified, kinto will try to open the IDB with the Firefox
+   * experimental storage option. If it fails, it will retry without the option
+   * so your storage choice will basically be ignored in other browsers.
+   * For more information on the storage option, see
+   * https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Browser_storage_limits_and_eviction_criteria#Firefox_specifics
+   *
+   * @param  {String} dbname The database name.
+   * @param  {Object} options The database options.
    */
-  constructor(dbname) {
+  constructor(dbname, options = {}) {
```

Comment:
How can the options be passed from the outside? Please add some docs to mention the `permanent` option.